roles/openshift_buildoverrides/vars/main.yml

```diff
@@ -4,7 +4,7 @@ buildoverrides_yaml:
     configuration:
       apiVersion: v1
       kind: BuildOverridesConfig
-      forcePull: "{{ openshift_buildoverrides_force_pull | default('', true) }}"
+      forcePull: "{{ openshift_buildoverrides_force_pull | default('',true) | bool }}"
```

Comment:
untested, but I think this will allow for differentiation between unset and set values:
```
force_pull = openshift_build_overrides_force_pull | default('')
...

        forcePull: "{{ force_pull if force_pull == '' else force_pull | bool }}"
```